tests/downloads/qml/CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ set(QML_TESTS_LIBS
     UbuntuDownloadManager
 )
 
-find_program(XVFB xvfb)
+find_program(XVFB Xvfb)
```

Comment:
This seems like the wrong solution, as xvfb does not seem to actually be used anywhere, and is not in Build-Depends. Instead, we should probably remove this check and the if/else blocks below.

Reply:
Also true